dataland-frontend/src/components/forms/parts/elements/derived/ProcurementCategoryFormElement.vue

```diff
@@ -0,0 +1,156 @@
+              data-test="directSuppliersHeader"
+            />
+          </div>
+          <FormKit
+            type="group"
+            name="numberOfSuppliersPerCountryCode"
+            v-model="numberOfSuppliersPerCountryCodeValue"
+            label="Suppliers Per Country"
+          >
+            <div v-for="el in selectedCountries" :key="el.label">
+              <div class="justify-content-between flex align-items-center" data-test="supplierCountry">
+                <h5>{{ getCountryNameFromCountryCode(el.value) }}</h5>
+                <div class="justify-content-end flex align-items-center">
+                  <FormKit
+                    type="number"
+                    :name="el.value"
+                    min="1"
+                    step="1"
+                    validation="required"
+                    validation-label="Number of suppliers per country"
+                    data-test="supplierCountryValue"
+                    outer-class="my-0 mx-3"
+                  />
+                  <PrimeButton
+                    icon="pi pi-times"
+                    rounded
+                    class="p-button-icon"
+                    data-test="removeElementBtn"
+                    @click="removeItemFromListOfSelectedCountries(el.value)"
+                  />
+                </div>
+              </div>
+            </div>
+          </FormKit>
+        </div>
+      </div>
+    </FormKit>
+  </div>
+</template>
+
+<script lang="ts">
+import { defineComponent } from "vue";
+import InputSwitch from "primevue/inputswitch";
+import { BaseFormFieldProps } from "@/components/forms/parts/fields/FormFieldProps";
+import { FormKit } from "@formkit/vue";
+import UploadFormHeader from "@/components/forms/parts/elements/basic/UploadFormHeader.vue";
+import MultiSelectFormFieldBindData from "@/components/forms/parts/fields/MultiSelectFormFieldBindData.vue";
+import NaceCodeFormField from "@/components/forms/parts/fields/NaceCodeFormField.vue";
+import PercentageFormField from "@/components/forms/parts/fields/PercentageFormField.vue";
+import PrimeButton from "primevue/button";
+import { DropdownDatasetIdentifier, getDataset } from "@/utils/PremadeDropdownDatasets";
+import { LksgProcurementCategory } from "@clients/backend";
+import { getCountryNameFromCountryCode } from "@/utils/CountryCodeConverter";
+
+export default defineComponent({
+  name: "ProcurementCategoryFormElement",
+  inject: {
+    procurementCategories: {
+      from: "procurementCategories",
+      default: {} as { [key: string]: LksgProcurementCategory },
+    },
+  },
+  components: {
+    InputSwitch,
+    FormKit,
+    UploadFormHeader,
+    MultiSelectFormFieldBindData,
+    PrimeButton,
+    NaceCodeFormField,
+    PercentageFormField,
+  },
+  props: BaseFormFieldProps,
+  data() {
+    return {
+      isItActive: !!this.procurementCategories[this.name],
+      procuredProductTypesAndServicesNaceCodesValue: [],
+      percentageOfTotalProcurementValue: "",
```

Comment:
If you type this as number, you won't need the "String" type for the prop `percentageFieldValueBind` of the `PercentageFormField.vue`-component anymore.
Or would this cause problems?

Reply:
![image](https://github.com/d-fine/Dataland/assets/96523011/62e51450-acd7-47ce-bb43-c15ae4e1c393)
Component PercentageFormField.vue has an input of text type, we can change it to a number but then it will add such arrows to the input and it will be a different input

Reply:
Ah ok yes sorry. You are right of course.

But then it could be only of type "String", right?   Or will this break smth.?